src/Tribe/RSVP.php

```diff
@@ -991,8 +994,8 @@ public function delete_ticket( $event_id, $ticket_id ) {
 		$rsvp_options    = $this->tickets_view->get_rsvp_options( null, false );
 		$attendee_status = get_post_meta( $ticket_id, self::ATTENDEE_RSVP_KEY, true );
 
-		$adjustment = isset( $rsvp_options[ $attendee_status ]['decrease_stock_by']  )
-			? absint( $rsvp_options[ $attendee_status ]['decrease_stock_by'] )
+		$adjustment = isset( $rsvp_options[ $attendee_status ][ 'decrease_stock_by' ]  )
```

Comment:
spaces around array key in `$rsvp_options[ $attendee_status ]` can remain here, but not in `['decrease_stock_by']`